__tests__/secret.test.js

```diff
@@ -241,6 +242,39 @@ describe('Tozny identity client', () => {
     expect(latestVersion.exists).toBe(false)
   })
 })
+it('can create a secret and share it with a username', async () => {
```

Comment:
let's have another test for sharing with a username that doesn't exist and that it fails smoothly